benchmarks/luafun/bench.lua

```diff
@@ -0,0 +1,178 @@
+local fun = require 'fun'
+local jitlog_capnp = require "jitlog_capnp"
+local bor = bit.bor
+local lshift = bit.lshift
+-- Enable table.new function
+require("table.new")
+
+local luafile = io.open("benchdata/jitlog.bin", "rb")
+local text = luafile:read("*all")
+luafile:close()
+
+local jitlog = jitlog_capnp.JITLog.parse(text)
+local traces = jitlog.traces
+local functions = jitlog.functions
+
+local stats = {
+    top_aborted = false,
+    top_stitched = false,
+}
+
+function init()
+    for i, func in ipairs(functions) do
+        func.starts = 0
+        func.stops = 0
+        func.aborts = 0
+        func.sideexits = 0
+        func.started_aborts = 0
+        func.stitch_roots = 0
+    end
+
+    -- Trace id remapping table
+    local idmap = {}
+    local tracecount = 1
+
+    for i, trace in ipairs(traces) do
+        --Build start and stop location keys for all the traces

```

Comment:
Space after `--` to match other instances?

Reply:
fixed